src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/ContactDiaryStoreV2Tests.swift

```diff
@@ -0,0 +1,101 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+import FMDB
+import Combine
+@testable import ENA
+
+class ContactDiaryStoreV2Tests: XCTestCase {
```

Comment:
Please add a migration test.

1. Create the old schema
2. Write data into the database created by the old schema
3. Create new store
4. Execute migration
5. Test if the data is still there after the migration.